src/main/java/sk/durixx/library/model/entity/Library.java

```diff
@@ -0,0 +1,121 @@
+package sk.durixx.library.model.entity;
+
+import jakarta.persistence.*;
+import sk.durixx.library.model.dto.LibraryDto;
+
+@Entity
+@Table(name = "library")
+public class Library {
+
+    @Id
+    @SequenceGenerator(
+            name = "id_sequence",
+            sequenceName = "id_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "id_sequence"
+    )
+    private long id;
+
+    @Column(
+            name = "name_of_library"
+    )
+    private String nameOfLibrary;
+
+    @Column(
+            name = "country"
+    )
+    private String country;
+
+    @Column(
+            name = "city"
+    )
+    private String city;
+
+    @Column(
+            name = "adress"
+    )
+    private String address;
+
+    public Library() {
+    }
+
+    public Library(Builder builder) {
+        this.id = builder.id;
+        this.nameOfLibrary = builder.nameOfLibrary;
+        this.address = builder.address;
+        this.city = builder.city;
+        this.country = builder.country;
+    }
+
+    public static Library fromDto(LibraryDto dto) {
+        Builder builder = new Builder();
```

Comment:
Builder builder = new Builder(); is redundant variable use instead in one line
 return new Builder().withId...